lib/std/Io/net.zig

```diff
@@ -1333,6 +1333,8 @@ pub const Server = struct {
         /// Not enough free memory. This often means that the memory allocation is limited
         /// by the socket buffer limits, not by the system memory.
         SystemResources,
+        /// Socket is not listening for new connections.
```

Comment:
Can you replace this text, which is completely redundant with the error name, with a short summary of your nice bug report, which explains why encountering this error in practice is useful?